### ISSUE TITLE
weeks property doesn't handle negative days count

### ISSUE THREAD

**Issue by souliane** [dateutil/dateutil#459]
Hallo,

the weeks property is defined like that:
```
    @property
    def weeks(self):
        return self.days // 7

```
If you initialize with relativedelta(days=-1), the weeks property would return -1 instead of 0...

**Comment by pganssle** [comment]
Great catch! A PR fixing this would be welcome, otherwise I'll get on it as soon as I remember which rounding function has the right properties 😉

**Comment by pganssle** [comment]
Fixed by #460
